ee/se/modules/380-metallb/requirements/check.go

```diff
@@ -0,0 +1,45 @@
+/*
+Copyright 2024 Flant JSC
+Licensed under the Deckhouse Platform Enterprise Edition (EE) license. See https://github.com/deckhouse/deckhouse/blob/main/ee/LICENSE
+*/
+
+package requirements
+
+import (
+	"errors"
+
+	"github.com/deckhouse/deckhouse/go_lib/dependency/requirements"
+)
+
+const (
+	metallbConfigurationStatusKey             = "metallb:ConfigurationStatus"
+	metallbConfigurationStatusRequirementsKey = "metallbHasStandardConfiguration"
+)
+
+func init() {
+	checkRequirementConfigurationStatus := func(_ string, getter requirements.ValueGetter) (bool, error) {
+		configurationStatusRaw, exists := getter.Get(metallbConfigurationStatusKey)
+		if !exists {
+			return true, nil
+		}
+
+		configurationStatus := configurationStatusRaw.(string)
+		switch {
+		case configurationStatus == "nsMismatch":
```

Comment:
Better to use
switch configurationStatus := configurationStatusRaw.(string); configurationStatus
case "nsMismatch"
or
configurationStatus := configurationStatusRaw.(string)
switch configurationStatus
case "nsMismatch"